### ISSUE TITLE
Add login support

### ISSUE THREAD

**Issue by Vectorious** [magicalraccoon/tootstream#1]
I think we should use a [Click](http://click.pocoo.org/5/) prompt, so users can login either of the following ways:
```bash
$ tootstream --name "Foo" --pass "bar"
```

```bash
$ tootstream
Name: Foo
Pass: 

```

**Comment by Vectorious** [comment]
For #3 should we prompt for shard, defaulting if enter is pressed? Or just default unless explicitly specified:

```bash
$ tootstream --shard "foo.bar"
Name: Foo
Pass:

```

@magicalraccoon

**Comment by magicalraccoon** [comment]
Closed by 4f6e05e2fcc34d2464e8033bd2686857bd7f5f11
